packages/vuelidate/src/index.js

```diff
@@ -14,6 +14,22 @@ const VuelidateRemoveChildResults = Symbol('vuelidate#removeChiildResults')
  * @return {UnwrapRef<*>}
  */
 export function useVuelidate (validations, state, registerAs) {
+  if (!validations) {
+    const instance = getCurrentInstance()
+    if (instance.type.validations) {
+      const rules = instance.type.validations
+
+      state = computed(() => toRefs(reactive(instance.ctx)))
```

Comment:
Why 'reactive' then 'ref'?

Reply:
Just using `toRefs(instance.ctx)` works but it throws a warning that it expected a reactive object and not a plain one...

Reply:
Wont this work just fine?
```suggestion
      state = reactive(instance.ctx)
```

Reply:
I'm pretty sure I tried that and it did not. :(